### PR TITLE
Stage -> Prod Sync (February 2024 Release)

### PR DESCRIPTION
**Pull request by anthonypetersen**
Issues Addressed
-----
* [QOL/PROMIS Survey PWA/Trigger](https://github.com/episphere/connect/issues/877) - @anthonypetersen 
* [Add additional refusal options for QOL survey](https://github.com/episphere/connect/issues/881) - @anthonypetersen 
* [Slow loading time for Reports (Biospecimen)](https://github.com/episphere/connect/issues/755) - @JoeArmani 
* [Data updates/cleanups related to notification functionality](https://github.com/episphere/connect/issues/871) - @we-ai 
* [Send Magic Link Email with NCI email server](https://github.com/episphere/connect/issues/731) - @jhflorey 
* [SMS data import from Twilio](https://github.com/episphere/connect/issues/886) - @jhflorey 
* [Fetch sha (commit history) for existing surveys](https://github.com/episphere/connect/issues/906) - @JoeArmani

### CODE REVIEW

**Comment by we-ai** [comment]
My updates were tested in stage, and are ready for prod.

**Review by we-ai — APPROVED** [review]

**Comment by anthonypetersen** [comment]
changes approved and ready for prod

**Comment by jhflorey** [comment]
My changes ready for prod

**Review by jhflorey — APPROVED** [review]

File: config/prod/.env.yaml [hunk 1]
```diff
@@ -6,4 +6,8 @@ SG_FROM_NAME: Connect for Cancer Prevention Study
 SG_FROM_EMAIL: donotreply@myconnect.cancer.gov
 TWILIO_ACCOUNT_SID: projects/nih-nci-dceg-connect-prod-6d04/secrets/twilio-account-sid/versions/1
 TWILIO_AUTH_TOKEN: projects/nih-nci-dceg-connect-prod-6d04/secrets/twilio-sms-auth-token/versions/1
-TWILIO_MESSAGING_SERVICE_SID: projects/nih-nci-dceg-connect-prod-6d04/secrets/twilio-messaging-service-sid/versions/1
\ No newline at end of file
+TWILIO_MESSAGING_SERVICE_SID: projects/nih-nci-dceg-connect-prod-6d04/secrets/twilio-messaging-service-sid/versions/1
+APP_REGISTRATION_TENANT_ID: projects/nih-nci-dceg-connect-stg-5519/secrets/app-registration-tenant-id/versions/1
+APP_REGISTRATION_CLIENT_ID: projects/nih-nci-dceg-connect-stg-5519/secrets/app-registration-client-id/versions/1
+APP_REGISTRATION_CLIENT_SECRET: projects/nih-nci-dceg-connect-stg-5519/secrets/app-registration-client-secret/versions/1
```
Comment (JoeArmani) [inline config/prod/.env.yaml:12]: This could be an issue: stage path in prod env file.

**Review by JoeArmani — APPROVED** [review]
My changes are tested and approved in stage. Ready for prod.

**Pull request merged**